dawn/src/dawn/CodeGen/CXXNaive-ico/ASTStencilBody.cpp

```diff
@@ -210,7 +210,8 @@ void ASTStencilBody::visit(const std::shared_ptr<iir::FieldAccessExpr>& expr) {
       // accessName));
     }
   } else {
-    ss_ << "m_" << getName(expr) << "(" << argName_ << ")";
+    ss_ << "m_" << getName(expr) << "(" << argName_ << ","
+        << "k + " << expr->getOffset().verticalOffset() << ")";
```

Comment:
Minor thing: Can you print "k+" without spaces? Then it is consistent with the other backends iI think.